config/prod.js

```diff
@@ -25,4 +25,8 @@ module.exports = {
       },
     },
   },
+  newrelic: {
+    licenseKey: 'c53c018d69',
```

Comment:
for the record, this (which ends up in the HTML page) is different from the full license key that APM and other products use. Not sensitive I'd say